tests/src/model/pages/dashboard-page.ts

```diff
@@ -17,17 +17,23 @@
  ***********************************************************************/
 
 import type { Locator, Page } from 'playwright';
-import { PodmanDesktopPage } from './base-page';
+import { BasePage } from './base-page';
 
-export class DashboardPage extends PodmanDesktopPage {
+export class DashboardPage extends BasePage {
```

Comment:
Should this extend MainPage?

Reply:
nice catch. although the answer could be no, imho. dashboard page is different than all others I am trying to pull logic together (images, containers, etc.) They share also searching, additional actions and possible table inside content part which Dashboard handle in a different way. There might be another abstract page between Base and Main page, but I am lacking naming ideas and it seems just adding another level in hierarchy. So the question is it worthy to add another level due to one page? I am not against, just having this argument against. :)

Reply:
Up to you tbh, i just commented because i saw something in common with the main page and i was not sure if it was made on purpose or not. You could also move the common thing between dashboard and main to base, if possible.

Reply:
Unfortunately BasePage cannot contain more since other types of pages do no shared that functionality (Like resources, or extensions pages). Maybe only thing that is shared is heading. But I will find out when adding more of those pages.

Reply:
although, it really seems that some functionality could be shared by all pages in base page, header and content parts. I would tackle that in different PR though, since we should then updated all the pages with particular role and label.